app/models/course/assessment.rb

```diff
@@ -38,6 +38,8 @@ class Course::Assessment < ActiveRecord::Base
   has_many :assessment_conditions, class_name: Course::Condition::Assessment.name,
                                    inverse_of: :assessment, dependent: :destroy
 
+  scope :published, -> { where(draft: false) }
```

Comment:
Think it would be useful to have a published scope. rails has provided a `draft` scope, but cannot be used here.